### DISCUSSION TITLE
Hello from Open Neuroscience Foundation

### DISCUSSION THREAD

**Discussion by nikolaydubina** [neurodesk/neurodesk.github.io-hugo#415] [General]
Hello,

I would like to update you that Neurodesk has been featured on https://neuroscience-landscape.com/. This is a major upcoming project for discovery of neuroscience resources.

If you see any updates need, such as description, logo, citations, h-index, links, simply open a Pull Request to this [repo](http://github.com/open-neuroscience-foundation/neuroscience-landscape), open an Issue, or contact me. If you have any feedback or would like to improve anything in this system, feel free to send it to me.

For now I added only one project there [neurocommand](https://neuroscience-landscape.com/?item=software--copmutation--neurocommand). I will try to add more later on when I finish other updates. Feel free to updated it with other projects!

Thank you once again for all the great you have been doing for advancement of neuroscience and I hope this project will put you on a map and give recognition it deserves!

**Comment by stebo85** [comment]
Awesome, that's great @nikolaydubina  :)
